arlas-tests/src/test/java/io/arlas/server/CollectionTool.java

```diff
@@ -42,23 +43,28 @@
     public static void main(String[] args) throws IOException {
         switch (args[0]) {
             case "load":
-                load(0);
```

Comment:
Why this is no longer static?

Reply:
Because it is also a test. It must NOT be static.

Reply:
Maybe that's the point : is it really a `@Test` method?
IMO, `CollectionTool` should only provide static methods.

Reply:
It was a @Test but JUnit was not happy about it being static. Can be removed in the future if considered useless.